uber/site_sections/registration.py

```diff
@@ -901,3 +901,6 @@ def inactive(self, session):
                                 .filter(~Attendee.badge_status.in_([c.NEW_STATUS, c.COMPLETED_STATUS]))
                                 .order_by(Attendee.badge_status, Attendee.full_name).all()
         }
+
+    def error_attendee_not_found(message):
+        return "Sorry, but that attendee doesn't exist."
```

Comment:
Two minor things:

1) The parameter to this function should be named ``self``, which is the Python version of ``this`` from other languages such as C / C++ / Java / Javascript / etc.  The first parameter to any instance method such as this will automatically be the class instance, which by convention you should always name ``self`` unless you have a really good reason not to.

2) As Dom already mentioned, our style checker requires newlines at the end of each file (which is what the PEP-8 style guide recommends), so you'll have to change your editor / IDE to do that.

Slightly more important question:

This page handler is currently in ``registration.py``, which is a page which is restricted to admins.  I'm not sure whether this is an error page we'll ever want to use for actions taken by non-admins, but if it is then we probably want it somewhere else where anyone can access it.  If we decide to do that, then ``preregistration.py`` is probably a good place for it.